fms/modules/attention.py

```diff
@@ -240,7 +250,7 @@ def forward(
 
         # if use_cache=True, we return the hidden_state as well as the kv cache
         if use_cache:
-            return out, (keys, values)
+            return out, (keys, values) if not fat_cache else (past_key_value_state[0], past_key_value_state[1], new_keys, new_values)
```

Comment:
given that black will already split this line into a few, might as well type down the whole if-else